src/code/bank1.asm

```diff
@@ -2601,13 +2601,13 @@ jr_001_6BC6::
     ldh  [hBGTilesLoadingStage], a
     ret
 
-Data_001_6BCF::
-    db $F, $51, $B1, $EF, $EC, $AA, $4A, $C
+Data_001_6BCF::  ; Instrument on inventory menu position ($9D00 is added to get to background position)
```

Comment:
I think that when you document a data label with comments, you could rename the data label as well. Even if the label name is not perfect, it's better than a `Data_<numbers>` label :)

Reply:
Yeah, sorry, I couldn't come up with a good name. And I was going all over the code trying to check what certain things are doing. The whole "from weapon to damaging entity" code is huge with all kinds of exceptions coded everywhere. I also had two different versions with notes laying around, and this one compiled and merged cleanly, while my other one is a huge mess. So figured I quickly push this to at-least not lose these notes as well.